README.md

```diff
@@ -15,6 +15,10 @@ If you know M-BoF, you don't need to know anything else.
 # Development
 0. Initialize and start app server
     - `cd /vagrant`
+    - `mysql -u root -p`
+    - Enter the password found in the Vagrantfile
+    - `mysql> CREATE DATABASE OCELLUS;`
+    - `mysql> exit`
```

Comment:
Some thoughts on this:

1. The provisioning section of Vagrantfile could do this for us.  We wouldn't need to type these commands at all.  For the QA environment, Docker can probably do it.
2. If you really don't want vagrant to do this and want to keep it as a command in this document, I believe this can be written as a single command.